rubygem-katello.spec

```diff
@@ -195,8 +195,18 @@ cp -a .%{gem_dir}/* \
 
 mkdir -p ./usr/share
 cp -r %{foreman_dir} ./usr/share || echo 0
+sed -i '/production:/,$d' ./usr/share/foreman/config/database.yml
```

Comment:
I think you need to create database.yml first by copying https://github.com/theforeman/foreman/blob/develop/config/database.yml.example to database.yml